ssp

```diff
@@ -1813,6 +1816,11 @@ sub check_perl_version_less_than_588 {
     }
 }
 
+sub print_time {
```

Comment:
The previous one-line 'uptime' output has been replaced with three new lines of terminal output and about 20 new lines of source that make up print_time, print_uptime, and print_load_average to avoid one execution of the external 'uptime' binary.  Is the added terminal output and code complexity worth shaving off a few milliseconds?  If it is worth avoiding the binary can we combine these all into a single uptime-equivalent line of output?

Reply:
I think we can definitely put it all into one line.